src/agent.h

```diff
@@ -382,12 +382,17 @@ class Agent : public StateWrangler, virtual public Ider {
   /// the agent has never been built).
   inline const int enter_time() const { return enter_time_; }
 
-  /// Sets the number of time steps this agent operates between building and
+  ///Sets the number of time steps this agent operates between building and
   /// decommissioning (-1 if the agent has an infinite lifetime).  This should
   /// generally only be called BEFORE an agent is added to a context as a
   /// prototype.  Throws ValueError if the agent has already been deployed.
   void lifetime(int n_timesteps);
 
+  /// Sets the number of time steps this agent operates between building and
+  /// decommissioning (-1 if the agent has an infinite lifetime).  The boolean
```

Comment:
what "boolean" are you referring to in the comment?

Reply:
Ahh, thanks. I was originally using a boolean but couldn't get it to work with the python bindings. I'll update that.